package.json

```diff
@@ -80,5 +80,8 @@
   },
   "engines": {
     "node": ">=12.0.0"
+  },
+  "dependencies": {
```

Comment:
shouldn't this be in the atomic package.json?

Reply:
Yup !